server/certidp/certidp.go

```diff
@@ -52,7 +52,14 @@ var (
 	}
 )
 
+// GetStatusAssertionStr returns the corresponding string representation of the StatusAssertion.
 func GetStatusAssertionStr(sa int) string {
+	/*
```

Comment:
Small nit but prefer // comments for consistency with the rest of the codebase.

Reply:
Ok, fixed it